### PR TITLE
Update doc for mxnet.ndarray.elemwise_add

### PR DESCRIPTION
**Pull request by indhub**
@mli, @zackchase, @madjam, @nswamy, @Roshrini, @jiajiechen
Please review and merge.

### CODE REVIEW

File: src/operator/tensor/elemwise_binary_op_basic.cc [hunk 1]
```diff
@@ -10,6 +10,16 @@ namespace mxnet {
 namespace op {
 MXNET_OPERATOR_REGISTER_BINARY(elemwise_add)
 .add_alias("_add").add_alias("_plus").add_alias("_Plus")
+.describe(R"code(Adds arguments element-wise.
+
+Example::
+
+  >>> x = mx.nd.array([1, 2, 3, 4])
```
Comment (piiswrong) [inline src/operator/tensor/elemwise_binary_op_basic.cc]: @mli Are we making all examples python specific or is there plans to add example for other languages? How do we separate them?

Comment (mli) [inline src/operator/tensor/elemwise_binary_op_basic.cc]: we should not add python examples in .cc file.

Comment (indhub) [inline src/operator/tensor/elemwise_binary_op_basic.cc]: @mli @piiswrong : I don't see this operator defined in the Python files. Any idea how to keep the description in the cc file and add the example elsewhere specific to Python?

Comment (nswamy) [inline src/operator/tensor/elemwise_binary_op_basic.cc]: @indhub, python specific examples should go to ndarray_doc.py. Also this does not have to be specific to Python.

File: python/mxnet/ndarray_doc.py [hunk 2]
```diff
@@ -37,6 +37,17 @@ class ReshapeDoc(NDArrayDoc):
     (4L, 3L, 2L)
     """
 
+class elemwise_addDoc(NDArrayDoc):
```
Comment (nswamy) [inline python/mxnet/ndarray_doc.py]: Would it make sense to add this example in the C++ documentation without the python-ness?. 
I feel this is a simple example and does not have be repeated in every language binding.

Comment (indhub) [inline python/mxnet/ndarray_doc.py]: I'm not sure how to write this example in a language neutral way. Is there any other function that has a language neutral example that I can use as reference?

Comment (nswamy) [inline python/mxnet/ndarray_doc.py]: see if  [Reshape](http://mxnet.io/api/python/ndarray.html#mxnet.ndarray.Reshape) helps.

File: python/mxnet/ndarray_doc.py [hunk 3]
```diff
@@ -37,6 +37,17 @@ class ReshapeDoc(NDArrayDoc):
...
+    """
+    Example
+    -------
+
+    >>> x = mx.nd.array([1, 2, 3, 4])
+    >>> y = mx.nd.array([1.1, 2.1, 3.1, 4.1])
+    >>> mx.nd.elemwise_add(x, y).asnumpy()
```
Comment (piiswrong) [inline python/mxnet/ndarray_doc.py]: x + y also works. Should we document that?

File: python/mxnet/ndarray_doc.py (hunk 2 above)
Comment (indhub) [inline python/mxnet/ndarray_doc.py]: Moved example into ndarray_doc.py

File: python/mxnet/ndarray_doc.py (hunk 3 above)
Comment (indhub) [inline python/mxnet/ndarray_doc.py]: Does x+y internally call elemwise_add?

**Comment by piiswrong** [comment]
please fix lint

File: python/mxnet/ndarray_doc.py (hunk 3 above)
Comment (piiswrong) [inline python/mxnet/ndarray_doc.py]: yes

**Pull request merged**